app/serializers/v3/upgrade_path_item_serializer.rb

```diff
@@ -12,6 +12,6 @@ def available
     object.mirror?
   end
 
-  attributes :friendly_name, :shortname, :identifier, :version, :arch, :release_type, :base, :product_type, :free, :release_stage, :available
+  attributes :shortname, :identifier, :version, :arch, :release_type, :base, :product_type, :free, :release_stage, :available
```

Comment:
I think you have to keep that attribute to make it part of the serializing. With the current state of the branch I see empty lines in the output of `zypper migration` where the friendly name once was shown:

```
bash-4.3# zypper migration

Executing 'zypper  refresh'

Repository 'SLES12-SP3-Pool' is up to date.                                                                                                                                                                          
Repository 'SLES12-SP3-Updates' is up to date.                                                                                                                                                                       
All repositories have been refreshed.

Executing 'zypper  --no-refresh patch-check --updatestack-only'

Loading repository data...
Reading installed packages...

Considering 0 out of 69 applicable patches:
0 patches needed (0 security patches)

Unavailable migrations (product is not mirrored):

         (not available)

         (not available)


No migration available.
```

I haven't mirrored fully, hence the note about "not available" ... but anyway, it doesn't show the name at all anymore.

@digitaltom Do we have an integration test for this?